src/clj/puppetlabs/services/watcher/filesystem_watch_service.clj

```diff
@@ -0,0 +1,45 @@
+(ns puppetlabs.services.watcher.filesystem-watch-service
```

Comment:
nitpick. This looks inconsistent with other tk services. looks like they mostly start with
`puppetlabs.trapperkeeper.services.` and not `puppetlabs.services.`